src/components/FormModal.jsx

```diff
@@ -0,0 +1,52 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faXmark } from '@fortawesome/free-solid-svg-icons';
+
+export default function FormModal({
+	handleSubmit,
+	closeForm,
+	existingToken,
+	handleChange,
+	errorMessage,
+}) {
+	return (
+		<div className="bg-[rgba(0,0,0,0.4)] w-screen h-screen flex justify-center items-center fixed">
+			<form
+				onSubmit={handleSubmit}
+				className="bg-white h-64 flex flex-col items-start justify-center gap-4 rounded-md px-10"
+			>
+				<FontAwesomeIcon
+					icon={faXmark}
+					className="text-xl self-end cursor-pointer hover:text-main"
```

Comment:
@neshacascia  Adding margin-right so that the cross sits symmetrically at the corner.
```suggestion
					className="text-xl self-end cursor-pointer hover:text-main -mr-5"
```